.github/workflows/scripts/install-build-deps.sh

```diff
@@ -38,3 +38,12 @@ fi
 [ -n "$LLVM_REPO" ] && wget --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 5 $LLVM_REPO/releases/download/llvm-$LLVM_VERSION-ispc-dev/$LLVM_TAR
 tar xf $LLVM_TAR
 echo "${GITHUB_WORKSPACE}/bin-$LLVM_VERSION/bin" >> $GITHUB_PATH
+
+# Fetch llvm-mingw: version of llvm doesn't matter as building only needs the headers in it
```

Comment:
I don't think we need to include mingw headers at all. I looked into `builtins-c-cpu.cpp` and it has only one platform specific function (`num_cores`) with implementation both for windows and posix(?). If we target windows we need just use the windows specific one.

Moreover, I noted one strange discrepancy in `builtins-c-cpu.cpp`:
`num_cores` platform specific implementation and its dependency are guarded by different `ifdef`s., here https://github.com/ispc/ispc/blob/9c2e40ad9282d20ee7a725caf8271634c9887967/builtins/builtins-c-cpu.cpp#L38 and here https://github.com/ispc/ispc/blob/9c2e40ad9282d20ee7a725caf8271634c9887967/builtins/builtins-c-cpu.cpp#L160

I think both should be guarded in same way, i.e., `#include <unistd.h` under `#if !defined(_MSC_VER) && !defined(__MINGW32__)`

@ishland , was this the only place that required mingw headers?